doc/ansible/user-guide.md

```diff
@@ -73,21 +73,21 @@ An example Watches file:
 - version: v1alpha1
   group: foo.example.com
   kind: Foo
-  role: /opt/ansible/roles/Foo
+  role: roles/Foo
```

Comment:
We should probably make roles relative to the `$ANSIBLE_ROLES_PATH`, which we should default to `$CWD/roles` in the container. This would mean you just specify `role: Foo`

Reply:
Agreed, that would make this even simpler (though explicitly call it out in the documentation so people know how to override if needed—though I can't think of a reason to at this time).